src/mapml/layers/TemplatedFeaturesLayer.js

```diff
@@ -65,6 +65,10 @@ export var TemplatedFeaturesLayer =  L.Layer.extend({
               }
             }));
           };
+        if(map.getZoom() % this._template.step !== 0) {
```

Comment:
I'm curious if because we're calling the _onMoveEnd method here if we couldn't get rid of the call to _pullFeatuerFeed on line 72? Or would that unleash the hounds...

Reply:
The _pullFeatureFeed does not get called if the statement above happens

Reply:
_onMoveEnd calls _pullFeatureFeed eventually.  Just wondering if the conditions could be moved into _onMoveEnd, removing code / reducing complexity. perhaps not.

Reply:
I'm not sure why this was the case in the first place. Removing the _pullFeatureFeed from onAdd and leaving the map.fire('moveeend') to trigger _onMoveend as before does not seem to change anything.

Reply:
Can you put that code into another branch and tell me what the branch is? I'll look at it and if we agree it's good we can merge it into this branch to be included in the PR...

Reply:
https://github.com/ben-lu-uw/Web-Map-Custom-Element/pull/3